san/sanbase_graphql.py

```diff
@@ -15,9 +15,17 @@
         'query': 'dailyActiveAddresses',
         'return_fields': ['datetime', 'activeAddresses']
     },
-    'burn_rate': {
-        'query': 'burnRate',
-        'return_fields': ['datetime', 'burnRate']
+    'burn_rate': { # to be removed
+        'query': 'tokenAgeConsumed',
+        'return_fields': ['datetime', 'tokenAgeConsumed']
```

Comment:
Here the return field is changed which will also break existing code. I would prefer the old query to stay and just add a deprecation warning when burn_rate is invoked : https://github.com/santiment/sanpy/pull/49/files#diff-0f5457262476aa06c3885256259a709dR69
example:
https://github.com/santiment/sanpy/blob/1f48d7dd01bdec929feabe6dc62e5c24798fb95c/san/__init__.py#L25